test/support/transition_landing_page_steps.rb

```diff
@@ -6,32 +6,15 @@ module TransitionLandingPageSteps
   include GdsApi::TestHelpers::ContentItemHelpers
   include RummagerHelpers
 
-  def given_there_is_a_transition_taxon
-    stub_content_store_has_item(transition_taxon_path, content_item)
-  end
-
-  def transition_taxon_path
-    "/transition"
-  end
+  CONTENT_ID = "d6c2de5d-ef90-45d1-82d4-5f2438369eea".freeze
```

Comment:
What is this the content id of - the taxon? Can we make it clear?

Reply:
This is the `/transition` taxon (formerly of the parish of `/brexit`)
[Matches here](https://github.com/alphagov/search-api/blob/1f91a38234c2f8c88b43854a3d039d9c26e2a4bf/lib/govuk_index/presenters/common_fields_presenter.rb#L11)

Reply:
But yeah i agree, could leave these as a comment? Or a link? What's most likely to be updated if we have to change the name again?

Reply:
Can we call it `TRANSITION_TAXON_CONTENT_ID`, to make that clearer? Like we call the path `TRANSITION_TAXON_PATH` (below)?

Reply:
Good idea, I like `TRANSITION_TAXON_CONTENT_ID`. 👏